Assets/Scripts/Gameplay/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    Rigidbody2D body;
+
+    float horizontal;
+    float vertical;
+
+    public float speed = 5.0f;
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        body = GetComponent<Rigidbody2D>();
```

Comment:
you should add RequiresComponent to the top of your class for a RigidBody2D if you are going to assume it has one here on Start